Functionality_Tests/FunctionalityTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using OpenQA.Selenium.DevTools.V115.Page;
+
+namespace Functionality_Tests_Suit
+{
+    [TestFixture]
+
+    public class FunctionalityTests : BaseTest
+    {
+
+        [Test, Order(1)]
+
+        public void PriceSorting()
+        {
+            SuccessfulLogin();
+            var elementItemSorting = driver.FindElement(By.TagName("select"));
+            elementItemSorting.Click();
+            var elementPriceSorting = driver.FindElement(By.CssSelector("[value='lohi']"));
+            elementPriceSorting.Click();
+            var elementMinPrice = driver.FindElement(By.CssSelector("[class='inventory_list']:first-child"));
+            Assert.That(elementMinPrice.Text.Contains("7.99"));
+
+            var elementItemsList = driver.FindElements(By.ClassName("inventory_list"));
+            Assert.That(elementItemsList, Is.Ordered.Ascending);
+        }
+    
+        [Test, Order(2)]
+
+        public void RemoveFromCart()
+        {
+            AddProductToCart();
+            var elementShopingcartIcon = driver.FindElement(By.ClassName("shopping_cart_link"));
+            elementShopingcartIcon.Click();
+            var elementRemoveButton = driver.FindElement(By.XPath("//button[text() = 'Remove']"));
+            elementRemoveButton.Click();
+            var elementShopingcartIcon2 = driver.FindElement(By.ClassName("shopping_cart_link"));
+            Assert.That(elementShopingcartIcon2.Text.Contains('1'), Is.False);
```

Comment:
But what if, after deleting the item from the cart, the count of products will be not 0, but 2?) test will pass, but shouldn't.
You can add assert that the element with number is not displayed instead check that number 1 is not present.
![image](https://github.com/Linases/Selenium/assets/55551919/c09c7399-8d15-42e1-b183-4f7e8695e597)

Reply:
fixed, thank you for the useful help!